com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -1426,7 +1430,7 @@ internal void HandleApproval(ulong ownerClientId, bool createPlayerObject, uint?
 
                 if (createPlayerObject)
                 {
-                    var networkObject = SpawnManager.CreateLocalNetworkObject(false, 0, playerPrefabHash ?? NetworkConfig.PlayerPrefabHash, ownerClientId, null, position, rotation);
+                    var networkObject = SpawnManager.CreateLocalNetworkObject(false, playerPrefabHash ?? NetworkConfig.PlayerPrefabHash.Value, ownerClientId, null, position, rotation);
```

Comment:
`PlayerPrefabHash` is no longer nullable so we need to remove `.Value` part here.

```suggestion
                    var networkObject = SpawnManager.CreateLocalNetworkObject(false, playerPrefabHash ?? NetworkConfig.PlayerPrefabHash, ownerClientId, null, position, rotation);
```